pkg/cluster/cluster.go

```diff
@@ -67,3 +74,124 @@ func (ln *localNode) Observe(ckit.Observer) {
 func (ln *localNode) Peers() []peer.Peer {
 	return []peer.Peer{ln.self}
 }
+
+// Clusterer implements the behavior required for operating Flow controllers
```

Comment:
Can you remove the previous implementation so we don't have both? The old implementation isn't being used anywhere.